nmmo/core/env.py

```diff
@@ -34,6 +34,7 @@ def __init__(self,
     self.obs = None
 
     self.possible_agents = list(range(1, config.PLAYER_N + 1))
+    self.has_scripted_agents = False
```

Comment:
how about self.scripted_agents = set()

That way we can just loop over those, instead of all the agents

Reply:
Changed has_scripted_agents to scripted_agents, which is an ordered set and made necessary changes to use it. 

Thank you so much, David!